website/content/sdk/guides/intros/stocks/government.md

```diff
@@ -0,0 +1,167 @@
+| 10 | 2022-10-01 | IGF::OT::IGF                                                                                                                                                                                    | DEPARTMENT OF JUSTICE (DOJ)           |     0    |
+| 11 | 2022-10-01 | ANTENNA SITES 113 AND 143                                                                                                                                                                       | DEPARTMENT OF HOMELAND SECURITY (DHS) | 29099.6  |
+| 12 | 2022-10-01 | IGF::OT::IGF BO/AMERICAN TOWERS ASSET SUB DBA SPECTRASITE COMMUNICATIONS/ANTENNA/WIR/11-01-18>10-31-19                                                                                          | DEPARTMENT OF JUSTICE (DOJ)           |  7692.24 |
+| 13 | 2022-10-01 | TUCSON MOUNTAIN                                                                                                                                                                                 | DEPARTMENT OF THE TREASURY (TREAS)    | 10056.8  |
+| 14 | 2022-09-10 | TOWER SITE RENTAL  REQUEST IS SUBJECT TO THE AVAILABILITY OF FY2023 FUNDS                                                                                                                       | DEPARTMENT OF JUSTICE (DOJ)           |     0    |
+| 15 | 2022-05-21 | FISCAL YEAR 2021 ANTENNA SITE(S) #043, LEASE RENEWAL - SUBJECT TO AVAILABILITY OF FUNDS.                                                                                                        | DEPARTMENT OF HOMELAND SECURITY (DHS) |     0    |
+
+### Histcont
+
+Get the historic (from 2008) quarterly sum paid to a company by the US Treasury department.
+
+```python
+gov.histcont('AMT')
+```
+
+|    |   Qtr |   Year |   Amount |
+|---:|------:|-------:|---------:|
+|  0 |     4 |   2021 |   127282 |
+|  1 |     3 |   2021 |   564310 |
+|  2 |     2 |   2021 |   564645 |
+| 52 |     4 |   2008 |   110021 |
+| 53 |     3 |   2008 |   240877 |
+| 54 |     2 |   2008 |   907099 |
+| 55 |     1 |   2008 |   320620 |
+
+### Lobbying
+
+Find the records of a company's lobbying efforts.
+
+```python
+gov.lobbying('AMT', limit = 500)
+```
+
+|    | Date       | Ticker   | Client                     |   Amount | Issue                                  | Specific_Issue                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Please reduce the size of this table to e.g. `limit=3`.

Reply:
done